vlad.metelap/task1.rb

```diff
@@ -0,0 +1,30 @@
+class Task1
+
+  b = Array.new(3, Array.new(3, 0))
+  # b = [[0,0,0],[0,0,0],[0,0,0]]
+  b = b.map {|el| el.map {rand(-10..10)}}
+  positivearr = b.map {|el| el.map {|elm| elm > 0 ? elm : 0} }
+
+  p "Task 1"
+  p "Array 01: " + b.inspect
+  b.select {|el| el.select {|elm| puts "#{elm}" if elm > 0} }
```

Comment:
You already found positive numbers and assing them to `positivearr ` variable. Let's use it

Reply:
done